src/libraries/System.IO.Ports/src/System.IO.Ports.csproj

```diff
@@ -109,7 +109,7 @@
     <Compile Include="$(CommonPath)Interop\Windows\Kernel32\Interop.FileOperations.cs"
              Link="Common\Interop\Windows\Kernel32\Interop.FileOperations.cs" />
   </ItemGroup>
-  <ItemGroup Condition=" '$(TargetsLinux)' == 'true' ">
+  <ItemGroup Condition=" '$(TargetsLinuxOrBrowser)' == 'true' ">
```

Comment:
Looks like this project didn't get a configuration update.  This is another package case.  We'd need to look and see if the port APIs we call are supported on WASM.  My guess is no and we should let it get a notsupported assembly.

Reply:
Please make it unsupported